Sources/GRPCCore/Metadata.swift

```diff
@@ -130,6 +130,13 @@ public struct Metadata: Sendable, Hashable {
     self.elements = []
   }
 
+  /// Initialize Metadata from a Sequence of Elements
+  public init(from elements: some Sequence<Element>) {
```

Comment:
Because this is a "value preserving" init you should omit the argument label: `init(_ elements: some Sequence<Element>)`.

See the "Argument Labels" section of https://www.swift.org/documentation/api-design-guidelines/